improver/wind_calculations/wind_downscaling.py

```diff
@@ -429,37 +426,38 @@ def _calc_u_at_h(self, u_in, h_in, hhere, mask, dolog=False):
         hhere = np.ma.masked_less(hhere, 0.0)
         upidx = np.argmax(h_in > hhere[:, :, np.newaxis], axis=2)
         # loidx = np.maximum(upidx-1, 0) #if RMDI, need below
-        loidx = np.argmin(np.ma.masked_less(hhere[:, :, np.newaxis] -
-                                            h_in, 0.0), axis=2)
+        loidx = np.argmin(
+            np.ma.masked_less(hhere[:, :, np.newaxis] - h_in, 0.0), axis=2
+        )
 
         if h_in.ndim == 3:
-            hup = h_in.take(upidx.flatten() + np.arange(0, upidx.size *
-                                                        h_in.shape[2],
-                                                        h_in.shape[2]))
-            hlow = h_in.take(loidx.flatten() + np.arange(0, loidx.size *
-                                                         h_in.shape[2],
-                                                         h_in.shape[2]))
+            hup = h_in.take(
+                upidx.flatten()
+                + np.arange(0, upidx.size * h_in.shape[2], h_in.shape[2])
+            )
+            hlow = h_in.take(
+                loidx.flatten()
+                + np.arange(0, loidx.size * h_in.shape[2], h_in.shape[2])
+            )
         elif h_in.ndim == 1:
             hup = h_in[upidx].flatten()
             hlow = h_in[loidx].flatten()
         uup = u_in.take(
-            upidx.flatten() +
-            np.arange(0, upidx.size * u_in.shape[2], u_in.shape[2])) \
-            # pylint: disable=unsubscriptable-object
+            upidx.flatten() + np.arange(0, upidx.size * u_in.shape[2], u_in.shape[2])
+        )  # pylint: disable=unsubscriptable-object
```

Comment:
The pylint needs to be up a line.

Reply:
I've done a bulk change on these pylint ignores so they go on the line before the relevant code.